src/services/mail.service.js

```diff
@@ -31,7 +31,7 @@ class MailService {
 			theme: 'salted',
 			product: {
 				name: 'Visit Africa App',
-				link: `http://localhost:3000/api/auth/verify/activate?token=${token}`,
+				link: `http://localhost:3000/api/auth/activate?token=${token}`,
 			},
 		});
 		const emailTemplate = mailGenerator.generate(emailToSend);
```

Comment:
If there isn't anywhere you are calling this `emailTemplate` variable why not can't you remove it?

Reply:
I am using it and it is the email template i am sending